app/components/Image/ProfilePicture.tsx

```diff
@@ -1,10 +1,13 @@
-import type { UserEntity } from 'app/reducers/users';
+import type { User } from 'app/models';
 import CircularPicture from './CircularPicture';
 import type { ComponentProps } from 'react';
 
 type Props = Omit<
   {
-    user: UserEntity;
+    user: Pick<
+      User,
+      'profilePicture' | 'username' | 'profilePicturePlaceholder'
+    >;
     size: number;
     className?: string;
   } & ComponentProps<typeof CircularPicture>,
```

Comment:
Props suddenly became difficult to read ..

Reply:
I mean, kinda, but just because of how it's formatted.

This is the best way to do it, since you can reuse type definitions so that changes to the user model will be propagated also in the types. Also, this makes sense in the sense that you just want some specific fields of a User. Instead of defining a new user model with those fields explicitly.

I don't think it's hard to read at all if you know TS properly tbh.

Reply:
Sure, I think it's just a matter of getting used to it